### ISSUE TITLE
Where can I find Requests.php

### ISSUE THREAD

**Issue by nytche** [Element-34/PHPBrowserMobProxy#5]
Requests.php is required in client.php file, but it is not in Pear neither in Github, where can I find it?

**Comment by rajeshtaneja** [comment]
I was trying to find the same, but got stuck with this file. Any clues ?

**Comment by nytche** [comment]
yes, I made some tests using https://github.com/rmccue/Requests, dapting some calls it should work.

**Comment by rajeshtaneja** [comment]
Thanks @nicocans,

I used chartjes/php-browsermob-proxy to get it working.
